src/main/java/duke/command/method/ListCommandMethod.java

```diff
@@ -0,0 +1,13 @@
+package duke.command.method;
+
+import duke.Duke;
+import duke.command.Command;
+
+public class ListCommandMethod implements CommandMethod {
+    public static final String NAME = "list";
+
+    public void execute(Duke program, Command command) {
+        program.getUi().print("Tasks so far:");
+        program.getUi().print(program.getTaskList().toString());
```

Comment:
Call to toString() is unnecessary "program.getTaskList().toString()".

Reply:
`Ui#print()` takes in a `String`, not a `TaskList` or `Object`. That being said, I guess I can overload the method...